grease-cli/tests/llvm/abort.llvm.cbl

```diff
@@ -4,7 +4,7 @@
   (start start:
     (let g (resolve-global "abort"))
     (let h (load-handle Unit () g))
+;; next_line_must_fail()
```

Comment:
@RyanGlScott: I think the ability to write this helper feels especially cool, so I just wanted to brag about it 😁 

It checks that the output contains `Likely bug: unavoidable error`, followed by a string of the form `{file}:{line+1}` where `{file}` is the name of the test file in question (here `abort.llvm.cbl`) and `{line+1}` is the line in that file following the line that this function is called on.